vendor/k8s.io/cloud-provider-gcp/providers/gce/gce_loadbalancer.go

```diff
@@ -166,7 +166,7 @@ func (g *Cloud) EnsureLoadBalancer(ctx context.Context, clusterName string, svc
 		return nil, err
 	}
 
-	klog.V(4).Infof("EnsureLoadBalancer(%v, %v, %v, %v, %v): ensure %v loadbalancer", clusterName, svc.Namespace, svc.Name, loadBalancerName, g.region, desiredScheme)
+	klog.V(4).Infof("debug EnsureLoadBalancer(%v, %v, %v, %v, %v): ensure %v loadbalancer", clusterName, svc.Namespace, svc.Name, loadBalancerName, g.region, desiredScheme)
```

Comment:
We should either remove the `debug` prefix in all the logging calls or change the log level.

I suspect this is left over from debugging work, though :)

Reply:
oh right, but I forgot to revendor after cleaning up my work. Thanks for catching this. Should be fixed now!